src/test/java/org/terasology/replayTests/examples/templates/ReplayTestTemplate.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.replayTests.examples.templates;
+
+
+import org.junit.After;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.terasology.ReplayTestingEnvironment;
+import org.terasology.engine.GameThread;
+import org.terasology.recording.RecordAndReplayStatus;
+
+import java.util.function.BooleanSupplier;
+
+
+/**
+ * This is a template with comments to aid in the creation of a replay test.
+ * For more information about Replay Tests, see https://github.com/MovingBlocks/Terasology/wiki/Replay-Tests
+ * Replay tests should extend ReplayTestingEnvironment directly or indirectly.
+ */
+public class ReplayTestTemplate extends ReplayTestingEnvironment {
```

Comment:
I'd move this to the wiki instead of leaving it in the code base. It'll be easier and cleaner to explain the usage of `ReplayTestingEnvironment` in the wiki using pretty markdown rather than having to explain it in javadoc.